### PR TITLE
fix problem with quotation

### PR DESCRIPTION
**Pull request by paulobarcelosbr**
## Please check if the PR fulfills these requirements

- [ ] The pull request follow our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## What kind of change does this PR introduce?

_(Bug fix, feature, docs update, ...)_

## What is the current behavior?

_(You can also link to an open issue here)_

## What is the new behavior?

_(if this is a feature change)_

## Does this PR introduce a breaking change?

_(What changes might users need to make in their application due to this PR?)_

## Other information

_(You can also share a screenshot)_

### CODE REVIEW

**Review by arthurmr96 — APPROVED** [review]

File: src/components/CheckoutModal.tsx [hunk 1]
```diff
@@ -219,7 +219,7 @@ export function CheckoutModal({
                         Balance: {userBalanceEth} ETH
                       </Text>
                       <Text style={{ fontSize: '12px' }} type='secondary'>
-                        {usdConversion && formatDollar(Number(userBalanceEth) * usdConversion)}
+                        {usdConversion && formatDollar(Number(totalPrice) * usdConversion)}
```
Comment (arthurmr96) [inline src/components/CheckoutModal.tsx:222]: Is this correct? Below the balance should be quotation in dollar of it

File: src/components/CheckoutModal.tsx [hunk 2]
```diff
@@ -241,9 +241,6 @@ export function CheckoutModal({
                     <Text style={{ fontSize: '12px' }} type='secondary'>
                       {nftSymbol.length > 22 ? `${nftSymbol.slice(0, 22)}...` : nftSymbol}
                     </Text>
-                    <Text style={{ fontSize: '12px' }} type='secondary'>
-                      {usdConversion && formatDollar(Number(userBalanceEth) * usdConversion)}
-                    </Text>
```
Comment (arthurmr96) [inline src/components/CheckoutModal.tsx:246]: Wouldn't be the correct set the quotation here of the total price?

**Review by arthurmr96 — CHANGES_REQUESTED** [review]

File: src/components/CheckoutModal.tsx (hunk 1 above)
Comment (paulobarcelosbr) [reply src/components/CheckoutModal.tsx:222]: yes that's correct, quote must be the value of the nfts and not the user's balance

File: src/components/CheckoutModal.tsx (hunk 2 above)
Comment (paulobarcelosbr) [reply src/components/CheckoutModal.tsx:246]: no because this information is redundant on the screen, the removal was validated with vini

**Review by arthurmr96 — APPROVED** [review]

**Pull request merged**